serve/elements/cem-pf-v6-progress/cem-pf-v6-progress.css

```
@@ -0,0 +1,310 @@
+  .danger #status-icon {
+    animation: fade-in
+      var(--pf-t--global--motion--duration--fade--default, 0.3s)
+      var(--pf-t--global--motion--timing-function--default, cubic-bezier(0.25, 0.1, 0.25, 1));
+  }
+}
+
+/* Elements */
+
+#description {
+  grid-column: 1 / 2;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)